core/toolbox.js

```diff
@@ -344,12 +344,19 @@ Blockly.Toolbox.prototype.getSelectedCategoryName = function() {
   return this.selectedItem_.name_;
 };
 
+/**
+ * @return {string} The id of the currently selected category.
+ */
```

Comment:
For all new functions here: if they're intended to be public, mark that explicitly.  Otherwise mark them package or private.